_sass/components/_va-nav-category.scss

```diff
@@ -0,0 +1,58 @@
+// Category navigation AKA "piano keys"
+// pattern. These are the links at the bottom
+// of secondary-level pages.
+
+.va-nav-category {
+  color: $color-gray-dark;
+  list-style: none;
+  margin: 0 !important;
+  padding: 0;
+  
+  @media #{$small} {
+    margin: 0 -0.625rem !important;
+  }
+  
+  &::before, &::after {
+    clear: both;
+    content: " ";
+    display: table; 
+  }
```

Comment:
I am not sure which browsers we are trying to support be if we need to support IE6+ then we might want to add `zoom` to trigger `hasLayout`.